lib/crawler/keyword/queries/keyword_query.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Crawler.Keyword.Queries.KeywordQuery do
+  import Ecto.Query
+
+  alias Crawler.Keyword.Schemas.Keyword
+
+  def user_keywords_query(user_id), do: where(Keyword, [keyword], keyword.user_id == ^user_id)
```

Comment:
should we `order_by` the `inserted_at field desc`

Reply:
Fix in [e30ade7](https://github.com/vnntsu/elixir-ic/pull/36/commits/e30ade7853ad15da4e2ca9ca7f92f5a63f2ee6df)